src/core/config.py

```diff
@@ -44,11 +44,15 @@ def get_bool(setting: str, default: str = "False") -> bool:
     return get_string(setting, default) == "True"
 
 
-# Параметры логгера
-LOG_NAME = get_string("LOG_NAME")
-LOG_NAME = LOG_NAME if LOG_NAME is not None else "bot.log"
+# Параметры общей папки с данными
+DATA_PATH = BASE_DIR / "../data"
```

Comment:
```suggestion
DATA_PATH = BASE_DIR.parent / "data"
```
Предлагаю делать папку `.data`